lib/tests/api/auth.rs

```diff
@@ -38,6 +38,23 @@ async fn signup_scope() {
 	.unwrap();
 }
 
+#[tokio::test]
+async fn signin_kv() {
+	let db = new_db().await;
+	db.use_ns(NS).use_db(Ulid::new().to_string()).await.unwrap();
+	let user = Ulid::new().to_string();
+	let pass = "password123";
+	let sql = format!("DEFINE LOGIN {user} ON KV PASSWORD '{pass}'");
```

Comment:
Might be missing something but defining a user on KV is only supported for the new `DEFINE USER` statement right?

Reply:
good catch! I didn't run the tests for that file yet because apparently VSCode don't pick them up. I'm now fixing the tests in that file 👍